app/controllers/companies_controller.rb

```diff
@@ -4,9 +4,8 @@
 
 class CompaniesController < ApplicationController
   def index
-    @companies = Company.all
-      .select { |company| company.condition == 'company active' }
-      .map { |company| CompanyPresenter.new(company) }
+    params[:condition] = 'company active' unless params[:condition]
```

Comment:
I would move the knowledge about the fact that `condition` is `company active` by default to the `CompaniesSelector`.

Reply:
In the other words, if `CompaniesSelector` gets `{query: "alior", stock: "nc"}` (no `condition` provided) is should filter companies by `contidion: "company active"`.

You may want to add unit tests for `CompaniesSelector` service (file in `spec/app/lib/companies_selector_spec.rb`) and test this case there.